web/src/components/home/subcomponents/upcoming_events/messages.js

```diff
@@ -0,0 +1,13 @@
+const messages = {
+  atStudioM: "@ Studio M Ballroom, San Jose, CA",
+  viewOnFacebook: "View on Facebook",
+  dyosEntryFee: "Entry fee: $10-$40",
+  dyosMaskingRequirement: "Masks required during classes",
```

Comment:
why are you duplicating messages? We already have these in common/messages

Reply:
My bad, I meant to move these out of common, and I just forgot to delete them. Deleted them from common